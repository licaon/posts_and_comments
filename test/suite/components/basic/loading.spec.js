import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import Loading from 'src/components/basic/loading';

describe('Loading component', () => {
  const createComponent = (props) => {
    const wrapper = shallow(
      <Loading
        {...props}
      />);
    return toJson(wrapper);
  };
  it('should render component correctly with props', (done) => {
    const component = createComponent({ id: 1 });

    expect(component).toMatchSnapshot();
    done();
  });
});
