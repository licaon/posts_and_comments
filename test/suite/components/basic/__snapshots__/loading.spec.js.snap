// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading component should render component correctly with props 1`] = `
<CircularProgress
  centered={true}
  className="loader"
  determinateDashoffset={187}
  id={1}
  scale={1}
  style={Object {}}
/>
`;
